programs/account-compression/tests/address_merkle_tree_tests.rs

```diff
@@ -93,6 +101,47 @@ pub async fn create_address_merkle_tree_and_queue_account(
         .process_transaction(transaction.clone())
         .await
         .unwrap();
+
+    let merkle_tree = AccountZeroCopy::<account_compression::AddressMerkleTreeAccount>::new(
+        context,
+        address_merkle_tree_keypair.pubkey(),
+    )
+    .await;
+
+    assert_eq!(merkle_tree.deserialized().index, 1);
+    // TODO(vadorovsky): Assert fees.
+    assert_eq!(
+        merkle_tree.deserialized().associated_queue,
+        address_queue_keypair.pubkey()
+    );
+    assert_eq!(
+        merkle_tree.deserialized().next_merkle_tree,
+        Pubkey::default(),
+    );
+    assert_eq!(merkle_tree.deserialized().owner, payer.pubkey());
+    assert_eq!(merkle_tree.deserialized().delegate, Pubkey::default());
+
+    let merkle_tree = merkle_tree.deserialized().copy_merkle_tree().unwrap();
+
+    let init_value = BigUint::from_str_radix(FIELD_SIZE_SUB_ONE, 10).unwrap();
+    let mut indexed_array = IndexedArray::<Poseidon, usize, 2>::default();
+    let nullifier_bundle = indexed_array.append(&init_value).unwrap();
```

Comment:
we have an init method in  IndexedArray that does exactly this.
see https://github.com/Lightprotocol/light-protocol/blob/main/merkle-tree/indexed/src/array.rs#L163-L168